skills/geoparquet/Skill.md

```
@@ -0,0 +1,344 @@
+```bash
+export AWS_PROFILE=my-profile
+# or
+export AWS_ACCESS_KEY_ID=...
+export AWS_SECRET_ACCESS_KEY=...
+```
+
+---
+
+## Example Session
+
+User: "I want to work with this dataset: https://data.source.coop/example/buildings.fgb"
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)